src/qtiItem/helper/util.js

```diff
@@ -136,7 +136,7 @@ var util = {
                 }
 
                 //search inside each elements:
-                _.each(collection, function(elt) {
+                _.each(collection, function (elt) {
```

Comment:
https://lodash.com/docs/4.17.15#forEach
```suggestion
                _.forEach(collection, function (elt) {
```